launcher/container_runner_test.go

```diff
@@ -414,6 +417,71 @@ type idTokenResp struct {
 	Token string `json:"token"`
 }
 
+func TestFetchContainerImageSignatures(t *testing.T) {
+	ctx := namespaces.WithNamespace(context.Background(), "test")
+
+	testCases := []struct {
+		name           string
+		targetRepos    []string
+		wantLen        int
+		wantBase64Sigs []string
+	}{
+		{
+			name:        "fetchContainerImageSignatures success",
+			targetRepos: []string{"us-docker.pkg.dev/vegas-codelab-5/cosign-test/base", "us-docker.pkg.dev/vegas-codelab-5/cosign-test/base"},
+			wantLen:     2,
+			wantBase64Sigs: []string{
+				// Check signatures from the OCI image manifest at https://pantheon.corp.google.com/artifacts/docker/vegas-codelab-5/us/cosign-test/base/sha256:1febaa6ac3a5c095435d5276755fb8efcb7f029fefe85cd9bf3ec7de91685b9f;tab=manifest?project=vegas-codelab-5.
+				"MEUCIQDgoiwMiVl1SAI1iePhH6Oeqztms3IwNtN+w0P92HTqQgIgKjJNcHEy0Ep4g4MH1Vd0gAHvbwH9ahD+jlnMP/rXSGE=",
+				"MEUCIQDgoiwMiVl1SAI1iePhH6Oeqztms3IwNtN+w0P92HTqQgIgKjJNcHEy0Ep4g4MH1Vd0gAHvbwH9ahD+jlnMP/rXSGE=",
+			},
+		},
+		{
+			name:        "fetchContainerImageSignatures success with nil target repos",
```

Comment:
Can we add a case for a nonexistent repo and one that exists but doesn't have signatures?